In [6]:
import re
import time
from meituan_tools import Operation
import pandas as pd
import pyecharts

In [7]:
# 数据库连接
op = Operation()
conn = op.db_conn()
cur = conn.cursor()

+ **拿到数据先整体数值字段max,min看下异常数据，这些极值会影响最后统计结果<br />观察后发现人均价格和评分有部分为0，对最后统计没有意义，所以在这里我们需要这些数据进行剔除**

In [3]:
# 数据检查
sql = '''	select distinct 
	b.poi_id,b.title,b.avg_price,b.avg_score,b.comment_num,b.address from meituan_classify_info as a
	inner join meituan_shop_info as b on a.sub_id = b.sub_id and a.class_type = b.class_type
	where a.class_type = 1 and a.sub_id not in(24,393,395) and b.avg_price <> 0 and b.avg_score <> 0
	order by avg_price desc'''

df = pd.read_sql(sql,conn,index_col=['poi_id'])
df

title  avg_price  avg_score  \
poi_id                                                                 
162761320                                  黄公子     2665.0        5.0   
2718768                          十里洋场江景会所（外滩店）     1489.0        4.0   
1578636                                 MOOK酒吧     1086.0        5.0   
1580796                                   浦江二号      994.0        5.0   
2473301                             洋房火锅（新天地店）      903.0        5.0   
476286                                  菲芘3d酒吧      883.0        4.0   
158735513                   本色TureColor酒吧（外滩店）      846.0        5.0   
454005                                  宏豪浦江一号      819.0        5.0   
40673951                          原烹（金虹桥国际中心店）      790.0        5.0   
480763                                 罗斯福酒窖餐厅      789.0        4.3   
2730725                                    花籠味      770.0        4.5   
487672                   Pelham's西餐厅（外滩华尔道夫酒店）      761.0        5.0   
6112293                                    香然会      723.0        4.5   
1579872                   THE PLUMP OYSTER BAR      677.0        5.0   
64049059                              榭福屋（静安店）      661.0        4.5   
159436207                       FEELING CLUB酒吧      658.0        5.0   
1607949                              晓芹海参（鲁班店）      655.0        3.5   
91906668                                   浦江荟      635.0        5.0   
419773                                    M1NT      619.0        4.5   
4093597    壳里西餐厅Coquille Seafood Bistro（壳里西餐厅）      618.0        5.0   
81256882                  TheMEAT∙扒餐厅（浦东嘉里大酒店）      611.0        4.0   
83259397                             晓芹海参（长宁店）      590.0        3.5   
94392794                        Da Ivo 意大利魔镜餐厅      575.0        4.8   
138245571                         蟹的岡田屋（尚嘉中心店）      573.0        4.4   
403296                       翡翠36餐厅（浦东香格里拉大酒店）      570.0        3.5   
106050874                          SKY酒吧（同乐坊店）      567.0        5.0   
394974                         珍御轩（新加坡泰国村鱼翅酒楼）      562.0        4.0   
2661742                            外滩悦榕庄-酩缘中餐厅      561.0        3.5   
4973672                      MT 1839 Club（闵行店）      551.0        3.9   
2421173                            世博源公馆（世博源店）      550.0        4.7   
...                                        ...        ...        ...   
5415000                              珍奶工坊（堡镇店）        9.0        4.5   
4441687                            武汉热干面（民耀路店）        9.0        4.3   
41396641                             鸡翅包饭（松江店）        9.0        4.0   
89590945                       吉合烤饼（嘉杰国际商业广场店）        9.0        4.3   
99356478                         C.忆奶茶（江桥沃尔玛店）        9.0        4.9   
74268107                            一品鸡排（华江路店）        9.0        4.7   
2410561                           超级奶爸（江桥沃尔玛店）        9.0        4.1   
41461534                            和路雪星任冷饮批发部        9.0        5.0   
78705706                       秋满阿姨奶茶（华新宝龙广场店）        9.0        4.6   
5949468                              快乐柠檬（南门店）        9.0        5.0   
41356582                           阿姨奶茶（东门大街店）        9.0        4.4   
96769827                      正宗湖南长沙臭豆腐（州桥老街店）        9.0        5.0   
84978808                                闲趣小吃奶茶        8.0        5.0   
115394563                           正新鸡排（青湖路店）        8.0        4.5   
42067538                      阿姨奶茶桥梓湾绿港店（桥梓湾店）        8.0        3.8   
98142819                              乐华仕（兴塔店）        8.0        5.0   
82035969                                  英伦时光        8.0        5.0   
435280                            鲜茶柠檬宝贝（庙前街店）        8.0        3.8   
100932130                                 天龙花甲        8.0        5.0   
94145441                           阿姨奶茶（飞航广场店）        8.0        4.4   
73651050                                土耳其肉夹馍        8.0        3.5   
94186511                          阿姨奶茶（北翼商业街店）        8.0        5.0   
69767825                                台湾脆皮玉米        7.0        3.4   
88196929 

In [4]:
from pyecharts import Liquid

liquid = Liquid(title="样本数",subtitle='样本总量29876 剔除均价和评分为0的数据所剩的数量26793')
liquid.add("Liquid", [0.89, 0.7, 0.5, 0.3], is_liquid_outline_show=False)
liquid

In [5]:
df = df.head(10)
title = df['title']
avg_price =  df['avg_price']
avg_score = df['avg_score']
comment_num = df['comment_num']

from pyecharts import Bar,Grid

bar = Bar(title="餐饮商家前十排行",subtitle='数据来源神秘组织：*团',width=800,height=400)
bar.add("人均", title, avg_price, mark_point=["min", "max"],mark_line=["average"])
bar.add("评论数", title, comment_num,mark_point=["max"],is_label_show=True, xaxis_rotate=30)

grid = Grid(height=500)
grid.add(bar, grid_bottom="30%")

grid

+ **对人均降序排序后，发现最贵的是“黄公子”人均2665，不由得让我们对前10的商家起了兴趣，于是上了百度查看了一下**

>+ **[【黄公子】](http://s.dianping.com/topic/33883578?utm_source=forum_pc_index)<br />
隐藏在老式洋房里的一家人均千元的定制私房菜，仪式感十足，每天只接受10位顾客的预定，简友们谁家宽裕的可以带我去遛遛 =。=**

<div style="float:left;clear:both ;solid 000;margin:2px;">
    <img src="https://p1.meituan.net/dpgroup/089fb645a439d5112f54709464198ebc2258883.jpg" height="300" width="180"/>
</div>    
<div style="float:left;border:solid 1px 000;margin:2px;">
    <img src="https://p0.meituan.net/dpgroup/08c6410707c7ee013fe42f49367a0f873015366.jpg" height="300" width="180"/>
</div> 
<div style="float:left;border:solid 1px 000;margin:2px;">
    <img src="https://p1.meituan.net/dpgroup/0d0c119c8c5ed617733543dbc6c814571009636.jpg" height="300" width="180"/>
</div> 
<div style="float:left;border:solid 1px 000;margin:2px;">
    <img src="https://p1.meituan.net/dpgroup/b7be7517245ca663efe94ac6d9c52ed22526430.jpg" height="300" width="180"/>
</div> 
<div style="float:left;border:solid 1px 000;margin:2px;">
    <img src="https://p1.meituan.net/dpgroup/6285d273f31ab98e1cbe5061747095ad2247619.jpg" height="300" width="180"/>
</div> 

>+ **[【MOOK酒吧】](http://www.dianping.com/shop/5583053)<br />什么是“当当当当，当当当?” 天哪! 还有漂亮的小姐姐...~哇靠~仿佛恋爱了... =。=**

<div style="float:left;clear:both ;solid 000;margin:2px;">
    <img src="http://qcloud.dpfile.com/pc/fh2O7aK9LdOvLZsPRasECbn4uJsOpNAw0yd7AzdTGAf9FiMxfamhzOpBpUB36ZKETYGVDmosZWTLal1WbWRW3A.jpg" height="300" width="180"/>
</div>    
<div style="float:left;border:solid 1px 000;margin:2px;">
    <img src="http://qcloud.dpfile.com/pc/CvQummZpOhNfK0n2sPv8sFcmkPzo3yecpXgwr1Lrzd4kNVqbowOSz6WrD4VBGHSxTYGVDmosZWTLal1WbWRW3A.jpg" height="300" width="180"/>
</div> 
<div style="float:left;border:solid 1px 000;margin:2px;">
    <img src="http://qcloud.dpfile.com/pc/Kjehf2z1DpAYJShzXNEcud6V7W2TsUTmYRErB8FI7s44OnA797ve0qcViZcSKwMVTYGVDmosZWTLal1WbWRW3A.jpg" height="300" width="180"/>
</div> 
<div style="float:left;border:solid 1px 000;margin:2px;">
    <img src="http://qcloud.dpfile.com/pc/PMC7S3ty3JH3sw-xVP93x3TSsjarY5GxYx-gycoJzhsru7nMSCtlSj7iGU77jW6uTYGVDmosZWTLal1WbWRW3A.jpg" height="300" width="180"/>
</div> 
<div style="float:left;border:solid 1px 000;margin:2px;">
    <img src="http://qcloud.dpfile.com/pc/DdbjvXy730rUs9XFo8gqiZ29HqXj0m1tsQE87NFZxXMTsu-eL1pFIl8y1JR6Gd7YTYGVDmosZWTLal1WbWRW3A.jpg" height="300" width="180"/>
</div> 

>+ **[【洋房火锅】](http://www.dianping.com/shop/8844957)<br />火锅中的劳斯莱斯...A级和牛600~800,涮一片牛肉小两百...我的天...贫穷真的限制我了我的想象 =。=**

<div style="float:left;clear:both ;solid 000;margin:2px;">
    <img src="http://qcloud.dpfile.com/pc/1qJi9p-wLeonVl6Y9WiyFPSanre0MLkueCtCk25iI7w85m6KLxJrZMuRwDiePBFkTYGVDmosZWTLal1WbWRW3A.jpg" height="300" width="180"/>
</div>    
<div style="float:left;border:solid 1px 000;margin:2px;">
    <img src="http://qcloud.dpfile.com/pc/o1Jnmy2bPPChRikogYoVC-pw_btvmkBBrWfbZefBAN0kNVqbowOSz6WrD4VBGHSxTYGVDmosZWTLal1WbWRW3A.jpg" height="300" width="180"/>
</div> 
<div style="float:left;border:solid 1px 000;margin:2px;">
    <img src="http://qcloud.dpfile.com/pc/exfn_RFUtZ0im46Rl-MXHedTYeJbsQERCP5g57Wid75WzN_4hUPhHPENFzwE2h71TYGVDmosZWTLal1WbWRW3A.jpg" height="300" width="180"/>
</div> 
<div style="float:left;border:solid 1px 000;margin:2px;">
    <img src="http://qcloud.dpfile.com/pc/6KngG1CJCk6uC6Ym2VWxmCzc5JFFiQKAAXEcmK-fA_t2Z2NWo8mGzuxwarQuIULZTYGVDmosZWTLal1WbWRW3A.jpg" height="300" width="180"/>
</div> 
<div style="float:left;border:solid 1px 000;margin:2px;">
    <img src="http://qcloud.dpfile.com/pc/n_irq9QArm3UMjp0K2j5595shc1P2PGhOOx3gWR1N6gpJ36aly4bqURKSnWlKZwiTYGVDmosZWTLal1WbWRW3A.jpg" height="300" width="180"/>
</div> 

+ **看完最贵的，那么我们看下最热的，也就是评论数最多的**

In [6]:
sql3 = '''#sql3
select distinct a.sub_id,a.sub_name,
b.poi_id,b.title,b.avg_price,b.avg_score,b.comment_num,b.address from meituan_classify_info as a
inner join meituan_shop_info as b on a.sub_id = b.sub_id and a.class_type = b.class_type
where a.class_type = 1 and a.sub_id not in(24,393,395) and b.avg_price <> 0 and b.avg_score <> 0 
and CONCAT(b.sub_id,b.poi_id) not in ('6342030772','4050576755','6350576755','4052163162','2006068147006','5452800270','4087812358','6387812358','543311762')
order by b.comment_num desc limit 10;
'''
df3 = pd.read_sql(sql3,conn)
df3

sub_id sub_name    poi_id                   title  avg_price  avg_score  \
0      36     小吃快餐   4091974            方燕烤猪蹄店（罗宾森店）       10.0        4.5   
1      40      自助餐   5396841        多伦多海鲜自助餐厅（江桥万达店）       99.0        3.9   
2      40      自助餐  42030772  上海贻贝湾海鲜自助餐厅(百联南桥购物中心店)       95.0        3.7   
3      36     小吃快餐   5565604           西安章记（曹路宝龙广场店）       18.0        4.8   
4      36     小吃快餐   4137469             盘古烤猪蹄（鹿都总店）       13.0        4.8   
5      17       火锅  68147006        牛市缘潮汕鲜牛肉火锅（乐颂坊店）       92.0        3.5   
6      40      自助餐  52800270          炭香园自助烧烤（人民北路店）       57.0        3.5   
7      36     小吃快餐   1584738          粉面桃花土豆粉（天钥桥路店）       24.0        4.2   
8      40      自助餐   3311762           圣火营BBQ（南京东路店）       57.0        3.5   
9      40      自助餐  40833128  鑫海汇海鲜烤肉火锅自助餐厅（巴黎春天天山店）       70.0        3.3   

   comment_num                               address  
0        24465                嘉定区城中路138号罗宾森广场南侧2号美食亭  
1        20903  嘉定区华江路848号江桥万达广场百货楼5楼5001铺位（金运路地铁站东）  
2        19766                奉贤区百齐路588号百联南桥购物中心二期5层  
3        17251                浦东新区金海路2505号曹路宝龙广场一期1层  
4        15752                     松江区庙前街98号（面潮未来旁边）  
5        14511                      松江区松汇中路568号乐颂坊3楼  
6        13116                 松江区人民北路112号三楼（招商银行楼上）  
7        12774                    徐汇区天钥桥路93-3号（近美罗城）  
8        12276             黄浦区南京东路588号丝绸商厦6楼（世纪广场对面）  
9        12110                    长宁区天山路762号巴黎春天中区4层

+ **从数据表中可以看出，整个评论排行榜的前十均被 小吃快餐、自助餐 2类霸榜（其中还剔除了各种xxx分店），有趣的是前十店铺的地址大多都在 嘉定、奉贤、松江、曹路 这些地区都在上海外环以外，离市区是相当远，兄die!我想去吃吃验证下都心有余而力不足。最后这两类分类到底是否真的是客量大，还是存在刷榜，或是神秘组织\*团商铺分类关系，个人就不深度去研究了...太花费时间，但我真的很好奇~知道的简友们请留言...**

In [7]:
from pyecharts import Bar,Grid

data = sorted(df3[['title','comment_num']].values ,key=lambda x: x[1],reverse=True)
attr = [i[0] for i in data]
val = [i[1] for i in data]

bar = Bar()
bar.add('comment_num', attr, val,is_label_show=True,xaxis_rotate=30)
grid3 = Grid(height=500)
grid3.add(bar, grid_bottom="30%")

grid3

> [【燕烤猪蹄店】](http://www.dianping.com/shop/58919293)
>+ **评论数量第一的竟然是家烤猪蹄店，个人是超级喜欢吃猪蹄的，皮脆肉软还美颜...Duang!!但为何以如此高的评论数位居榜首？对这块了解的请留言告知~(喜欢图3的请为我点赞!)**

<div style="float:left;border:solid 1px 000;margin:2px;">
    <img src="http://qcloud.dpfile.com/pc/ktUEW5kKJiEidB7kDqF_j1a0z62NhOm5YlkD4uLuOaxF3HNB3vggzmw8raY1JNntTYGVDmosZWTLal1WbWRW3A.jpg" height="300" width="330"/>
</div>
<div style="float:left;border:solid 1px 000;margin:2px;">
    <img src="http://qcloud.dpfile.com/pc/Evr544EhAIuulJNnEolo1ADBfIGYusZQGC9aZIW_qG4LPJtbh_j3JRHoujrHxBzGTYGVDmosZWTLal1WbWRW3A.jpg" height="300" width="330"/>
</div>    
</div> 
<div style="float:left;border:solid 1px 000;margin:2px;">
    <img src="http://qcloud.dpfile.com/pc/Qc0poBNvPAog_0wiqSoRscCswJHzgdj1ZJ52VUVRRPJJZLFfMo2DngJBqjtlKhWfTYGVDmosZWTLal1WbWRW3A.jpg" height="300" width="300"/>
</div> 



+ **对餐厅感兴趣的可以点餐厅链接↑↑↑ <br /> <br />好了，言归正传，分类不同，价格会相差很大，所以不能对所有类别进行全量统计（例如日料和小吃的价格就是天壤之别）下面的箱线图就很好的表达了这些分类的数据分布情况（ps：箱线图可以表达很多基础分布,最上线表示最大，最下线表示最小，箱体内表示整体中正常数据，箱体外表示整体中部分突变数据，箱体中线表示中分为，箱体上框表示第一四分位，箱体下框表示第三四分位）**

In [8]:
from pyecharts import Boxplot

sql = '''select distinct sub_id,sub_name from meituan_classify_info where class_type = 1 and sub_id not in(24,393,395)'''
df = pd.read_sql(sql,conn)

sql2 = '''select distinct poi_id,avg_price,sub_id from meituan_shop_info where class_type = 1 and avg_price <> 0 and avg_score <> 0'''
df2 = pd.read_sql(sql2,conn)

In [9]:
x_axis = []
y_axis = []
for i in df.index:
    sub_id = df.loc[i].values[0]
    sub_name = df.loc[i].values[1]  
    avg_price = df2[df2['sub_id'] == sub_id ]['avg_price'].values
    x_axis.append(sub_name)
    y_axis.append(avg_price)
    
boxplot = Boxplot("菜系数据分布情况")
_yaxis = boxplot.prepare_data(y_axis)   # 转换数据
tp_dict = {k:v for k,v in zip(x_axis,_yaxis)}
idx_tp = sorted([(max(v),k) for k,v in zip(x_axis,_yaxis)])

# 根据max排序
x_ax = []
y_ax = []
for i in idx_tp:
    x_ax.append(i[1])
    y_ax.append(tp_dict[i[1]])
boxplot.add('boxplot', x_ax,y_ax,is_datazoom_show=True, datazoom_type='both',xaxis_rotate=30)

grid = Grid()
grid.add(boxplot, grid_bottom="20%")
grid

+ **分类有很多种，要把菜系和食品的类别区分开（火锅可以是川菜也可以是日料，日料可以是自助也可以是海鲜）所以我们要把分类再细化，这里剔除食品分类，筛选出菜系类别（如：日料、川菜、粤菜、浙江菜、西北菜等）**

In [10]:
sql4 = '''select sub_id,sub_name ,count(*) as cnt from (
	select distinct a.sub_id,a.sub_name,
	b.poi_id,b.title,b.avg_price,b.avg_score,b.comment_num,b.address,b.stamp from meituan_classify_info as a
	inner join meituan_shop_info as b on a.sub_id = b.sub_id and a.class_type = b.class_type
	where a.class_type = 1 and a.sub_id in (55,56,57,58,59,60,62,227,233,20003,20059,20060)
    and b.avg_price <> 0 and b.avg_score <> 0 
) as x
group by sub_id,sub_name;
'''
df4 = pd.read_sql(sql4,conn,index_col='sub_id')
df4

sub_name  cnt
sub_id              
55          川湘菜  796
56          江浙菜  653
57           粤菜  578
58          西北菜   84
59         京菜鲁菜   68
60          云贵菜   81
62         东南亚菜  202
227      台湾/客家菜   97
233         新疆菜   79
20003       东北菜  154
20059      日本料理  649
20060      韩国料理  373

>+ **下面就用饼图的变种南丁格尔玫瑰图皮一下，因为比较酷炫~ 但在实际应用中饼图并不能很好的诠释分类情况因为看面积有些近似的面积很难区分谁多谁少，还是柱状图来的直观**

In [11]:
from pyecharts import Pie

sorted_df = sorted(df4[['sub_name','cnt']].values, key=lambda x:x[1],reverse=True)
attr = [i[0] for i in sorted_df]
val = [i[1] for i in sorted_df]
    

pie = Pie("*团各大菜系店铺数", title_pos='center', width=800)

pie.add("菜系", attr, val, center=[50, 50], is_random=True,
        radius=[35,65], rosetype='radius',legend_orient='vertical',legend_pos='left',
        is_legend_show=True, is_label_show=True)
pie

+ **川湘菜、浙江菜、日料、粤菜、韩料 位居前5，可以说统计结果跟现实中完全吻合<br />继续对数据进行下钻，不才学过日语也了解过日本历史,所以对日料有着特别的青睐，接下来就来专门研究下日料的情况。**

>+ **插句题外话,若不是战争的原因日本是有着高度文明的一个国家，经常告诉自己四个字“工匠精神”，日本人捏饭团捏一辈子，凿冰凿一辈子，炸天妇罗炸一辈子，工匠们喜欢不断雕琢自己的产品，不断改善自己的工艺，享受着产品在双手中升华的过程。工匠们对细节有很高要求，追求完美和极致，对精品有着执着的坚持和追求，把品质从0提高到1，其利虽微，却长久造福于世，它是极高的职业道德、职业能力、职业品质的体现。<br /><br />ps：科普科普个小知识，IPAD研磨工艺就是出自5个人小作坊——[小林研业](http://www.360doc.com/content/18/0205/23/51477542_728005923.shtml)<br /><div style="float:left;border:solid 1px 000;margin:2px;">
    <img src="https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1523125122660&di=5a9e8b194a63dd60a6776c45c84ead23&imgtype=0&src=http%3A%2F%2Fimg.mp.itc.cn%2Fupload%2F20160825%2F04d18149b910402cb79928bb2038f0fe_th.jpg"/>
</div> 
<br />这家为苹果研磨iPod的企业，就是位于日本新潟县燕市的“小林研业”，正是在这间其貌不扬的厂房里5名研磨技师花费了大约4年时间，为广受全球顾客欢迎的超过100万个“iPod”进行了背板的镜面加工。**

In [12]:
sql5 = '''select distinct a.sub_id,a.sub_name,
b.poi_id,b.title,b.avg_price,b.avg_score,b.comment_num,b.address,b.stamp from meituan_classify_info as a
inner join meituan_shop_info as b on a.sub_id = b.sub_id and a.class_type = b.class_type
where a.class_type = 1 and a.sub_id = 20059 and b.avg_price <> 0 and b.avg_score <> 0;
'''
df5 = pd.read_sql(sql5,conn,index_col='sub_id')
df5

sub_name     poi_id                        title  avg_price  avg_score  \
sub_id                                                                          
20059      日本料理  152515863                       源季日本料理      115.0        5.0   
20059      日本料理     466334                          京都屋       78.0        3.0   
20059      日本料理  158829945                         牛繁烤肉      112.0        4.5   
20059      日本料理     397913                         伊万里川      282.0        5.0   
20059      日本料理    1556061              和惠自助烧肉精致料理（川沙店）      164.0        4.0   
20059      日本料理    5450114                       友楽日本料理      202.0        3.8   
20059      日本料理    6336794                    樱花いざかや居酒屋       86.0        4.0   
20059      日本料理   96210122                       匠品精致料理      282.0        4.6   
20059      日本料理     475185                    御寿司（国定路店）       25.0        4.5   
20059      日本料理  157229828                    井禾寿司（金杨店）       55.0        4.0   
20059      日本料理    5081601                   藤幸烧肉（愚园路店）      122.0        4.2   
20059      日本料理    1469295            知竹日式料理（恒基名人购物中心店）       41.0        3.3   
20059      日本料理   96812271                        晶之辉和食      150.0        4.5   
20059      日本料理  152078006                 千艾炭火烧肉屋.夜食酒场      101.0        4.1   
20059      日本料理  160291382                禾尔观海（金海宝龙广场店）       38.0        4.1   
20059      日本料理   85872848                  伊腾寿司（芷江西路店）       77.0        5.0   
20059      日本料理    1592492             食特美烧肉スタミナ苑（虹梅路店）      171.0        4.2   
20059      日本料理    6836298               清水日本料理（仙乐斯广场店）      238.0        3.6   
20059      日本料理    2456978              菊樱寿司（中冶祥腾城市广场店）       81.0        3.8   
20059      日本料理    4725587              上海绿地万豪酒店“红”日式料理      187.0        4.6   
20059      日本料理   87670084                       鱼真鲜鱼料理      322.0        4.7   
20059      日本料理   51091395                酷乐友居酒屋（天乐广场店）      108.0        4.0   
20059      日本料理   41426428                  東大麺王（百联西郊店）       47.0        5.0   
20059      日本料理    6396052                    大焰日本料理铁板烧      159.0        4.5   
20059      日本料理    2451694                 牛New寿喜烧（徐汇店）      109.0        5.0   
20059      日本料理    6654152                  上井精致料理（青浦店）      227.0        3.6   
20059      日本料理     380967                       謙八日本料理      274.0        4.0   
20059      日本料理   42664206                       三条日本料理      121.0        4.6   
20059      日本料理    4948441               筑地海幸三代目（马勒别墅店）      319.0        4.2   
20059      日本料理    6259805                炭宝贝烤肉（三号湾广场店）       89.0        4.2   
...         ...        ...                          ...        ...        ...   
20059      日本料理    1138628                      洛卡自助铁板烧       82.0        4.0   
20059      日本料理  158389219                         大吉烧肉      206.0        4.5   
20059      日本料理    6234044                      烧太郎日本料理      104.0        5.0   
20059      日本料理    1592049                  渔乐勝（乐胜日本料理）      139.0        3.7   
20059      日本料理    1600947                北海道日本料理（叶城路店）      102.0        4.2   
20059      日本料理   41900910                       红亭炭火烧肉      125.0        4.2   
20059      日本料理   92987538                       咚咖滋猪排饭       69.0        4.2   
20059      日本料理  120859247                富罟河豚料理（大宁国际店）      310.0        3.9   
20059      日本料理   75828548                 胜世日式猪排（湖滨道店）       68.0        3.5   
20059      日本料理  153727197                          竹月亭      154.0        4.5   
20059      日本料理  110718459                  和风铁板料理（共江店）       59.0        3.8   
20059      日本料理   50085417              渔夫铁板烧(罗宾森购物中心店)       36.0        4.0   
20059      日本料理   93611812            百合福·元丽日料放题（上海广场店）      473.0        3.9   
20059      日本料理     395498                      柚子林炭火烧肉      252.0        4.1   
20059      日本料理  112522424                    喜食屋（康桥路店）       76.0        4.3   
20059      日本料理   41617495                 泗喜屋·日料（丰庄路店）       60.0        4.5   
20059      日本料理   95616545            

<div style="float:left;clear:both ;solid 000;margin:2px;">
    <img src="http://qcloud.dpfile.com/pc/Rnmqu5D-Lc_Aj3XF6btESggR1j0zhuMR5MerxBOGvhZ6X-SGifYn7dkErBNhKP0ATYGVDmosZWTLal1WbWRW3A.jpg" height="300" width="180"/>
</div>    
<div style="float:left;border:solid 1px 000;margin:2px;">
    <img src="http://qcloud.dpfile.com/pc/grVIys6PnGRFd0KJXPM6D5L1cm2--ZIdVVrMzIafnrDVeCVtTYZQnZunqDYamz1ITYGVDmosZWTLal1WbWRW3A.jpg" height="300" width="180"/>
</div> 
<div style="float:left;border:solid 1px 000;margin:2px;">
    <img src="http://qcloud.dpfile.com/pc/iT5hKDR-SZXcZZe6MdqzWZ51dDb3dh1RkHFdtqxUNZ4Ho8itsBkcmVncTI8kIYxjTYGVDmosZWTLal1WbWRW3A.jpg" height="300" width="180"/>
</div> 
<div style="float:left;border:solid 1px 000;margin:2px;">
    <img src="http://qcloud.dpfile.com/pc/f5sfzed5xqWpi4nm0DrEGhG4ZRZ-GRLhizP3zjGSngZ3FBJMSLaCv8bY782i7uhfTYGVDmosZWTLal1WbWRW3A.jpg" height="300" width="180"/>
</div> 
<div style="float:left;border:solid 1px 000;margin:2px;">
    <img src="http://qcloud.dpfile.com/pc/UnXnR771C3W3EpvMi66Ow7Bsg8wmgjaDHqC2QmaIPHujpO0oDC1nRw8deAzYxv-zTYGVDmosZWTLal1WbWRW3A.jpg" height="300" width="180"/>
</div> 

黄浦区江西中路328号（南京东路新世界大丸百货）
showLocation&&showLocation({"status":0,"result":{"location":{"lng":121.49295308929035,"lat":31.244728849293325},"precise":1,"confidence":80,"level":"道路"}})
